test/module-systems/deno-test.ts

```diff
@@ -0,0 +1,17 @@
+import { test } from "https://deno.land/std/testing/mod.ts";
+import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
```

Comment:
You're not aiming the version in here.
CI is running using `0.21.0` let's try using `/std@v0.21.0/` here?

Reply:
`/std@v0.21.0/` worked like a charm, thx!